setup.py

```diff
@@ -54,6 +94,8 @@
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
aren't all versions listed here supposed to go into the matrix for the pypi job?

Reply:
No idea, and not sure how to name wheels with the right py version